src/rules/operator-no-unspaced/index.js

```diff
@@ -159,7 +159,7 @@ export function calculationOperatorSpaceChecker({ root, result, checker }) {
 
       // Media queries
       if (item.name === "media" || item.name === "import") {
-        mediaQueryParser(item.params).walk(node => {
+        mediaQueryParser(item.params).walk("", (node) => {
```

Comment:
I added this change because there is a bug in `mediaQueryParser`'s `walk` method that causes it to throw an error when running unit tests and coverage reporting to fail.